extension/jmxmetricsextension/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
Please add the module in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/.github/dependabot.yml